### PR TITLE
Nearsighted Sensors Fix

### PR DESCRIPTION
**Pull request by Geevies**
* Nearsighted characters now have working vision on sensors / helm as long as they have their prescription glasses on.

### CODE REVIEW

**Comment by Geevies** [comment]
!review

**Comment by Geevies** [comment]
!bugfix

**Comment by listerla** [comment]
https://github.com/Aurorastation/Aurora.3/issues/14380

File: code/modules/overmap/ships/computers/ship.dm [hunk 1]
```diff
@@ -58,6 +58,7 @@ somewhere on that shuttle. Subtypes of these can be then used to perform ship ov
 	LAZYDISTINCTADD(viewers, WEAKREF(user))
 	if(linked)
 		LAZYDISTINCTADD(linked.navigation_viewers, WEAKREF(user))
+	ADD_TRAIT(user, TRAIT_COMPUTER_VIEW, ref(src))
```
Comment (NonQueueingMatt) [inline code/modules/overmap/ships/computers/ship.dm:61]: probably add this to camera view as well

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**